command/assets/example-short.nomad

```diff
@@ -2,6 +2,10 @@ job "example" {
   datacenters = ["dc1"]
 
   group "cache" {
+    network {
+      port "db" {}
+    }
+
```

Comment:
When running locally against a dev agent (either root or non-root) I get:

```
$ nomad run https://raw.githubusercontent.com/hashicorp/nomad/6875248c1934185e9cbbe1485dcde6e69db280e5/command/assets/example-short.nomad
Job Warnings:
1 warning(s):

* Group "cache" has warnings: 1 error occurred:
        * mbits has been deprecated as of Nomad 0.12.0. Please remove mbits from the network block



==> Monitoring evaluation "0acf50e0"
...
```

The task fails to run with:

````
Trying to map ports but no network interface is available
```

Reply:
:facepalm: I didn't update the examples with the new syntax. Fixed + added a more helpful error when `port_map` is used where `ports` should be